### PR TITLE
Rename Fork Specific Options From "DeltaV" To "EinsteinEngines"

### LINKED ISSUE #481: Fork specific options are still called "Delta-V" in the settings
# Media

image

### PR DESCRIPTION
**Pull request by Tmanzxd**
Fixes #481

### CODE REVIEW

File: Content.Client/Options/UI/OptionsMenu.xaml.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public OptionsMenu()
             Tabs.SetTabTitle(2, Loc.GetString("ui-options-tab-controls"));
             Tabs.SetTabTitle(3, Loc.GetString("ui-options-tab-audio"));
             Tabs.SetTabTitle(4, Loc.GetString("ui-options-tab-network"));
-            Tabs.SetTabTitle(5, Loc.GetString("ui-options-tab-deltav")); // DeltaV specific settings
+            Tabs.SetTabTitle(5, Loc.GetString("ui-options-tab-einsteinengines")); // DeltaV specific settings
```
Comment (DEATHB4DEFEAT) [inline Content.Client/Options/UI/OptionsMenu.xaml.cs:23]:
```suggestion
            Tabs.SetTabTitle(5, Loc.GetString("ui-options-tab-einsteinengines"));
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
I think this should just be removed, it's only the species vision right? We have traits for that.

File: Content.Client/Options/UI/OptionsMenu.xaml.cs (hunk 1 above)
Comment (Tmanzxd) [reply Content.Client/Options/UI/OptionsMenu.xaml.cs:23]: Yea we could remove the tab outright if the species vision is not applied.

**Comment by DangerRevolution** [comment]
> I think this should just be removed, it's only the species vision right? We have traits for that.

This goes back to the discussion about whether or not being able to cheaply opt out of species visions is okay or not. There's pros and cons to each side, but IMO it's for a different PR :)

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Until consensus about removing the tab or not, revert the removal :)

**Comment by Tmanzxd** [comment]
Got it

**Comment by VMSolidus** [comment]
My concession is that while yes it's important to have Accessibility Options, we currently don't have ANY other than the vision toggle(And most of the really good accessibility features require Harmony....), so can we move the button to one of the pre-existing tabs? Also, we can set that CVar to default to false.

**Comment by FoxxoTrystan** [comment]
Personnally il say move the option to another tab, simply.

**Comment by DangerRevolution** [comment]
Test fail valid

**Comment by Tmanzxd** [comment]
I can rebase the branch to the current master branch then do the tab rename if we want.

I can also remove the vision filter option as our traits cancel it out so the EE tab will be a placeholder for future additions.

**Comment by DangerRevolution** [comment]
> I can rebase the branch to the current master branch then do the tab rename if we want.
> 
> I can also remove the vision filter option as our traits cancel it out so the EE tab will be a placeholder for future additions.

Leave the options, just rename the tab.

**Comment by FoxxoTrystan** [comment]
Whats the update on that PR?

**Comment by VMSolidus** [comment]
Closed since it's superceded by https://github.com/Simple-Station/Einstein-Engines/pull/928

**Pull request closed without merging**